tests/e2e/src/pages/contacts.page.ts

```diff
@@ -49,25 +47,6 @@ export class ContactsPage extends BasePage {
     return `${this.contactsPageContent}//div[text()='${contactName}']`;
   }
 
-  async fillContactFields(inputFieldName: string, text: string) {
-    const helper = new Helper(this.world);
-    if (
-      inputFieldName == "Name of the contact" ||
-      inputFieldName == "Ethereum address" ||
-      inputFieldName == "Address or ENS or contact name" ||
-      inputFieldName == "Symbol or address"
-    ) {
-      selector = `//*[@placeholder="${inputFieldName}"]`;
-      if (text === "clipboard") {
-        const result = await helper.getClipboardValue();
-        await this.fill(selector, result);
-      }
-      await this.world.page?.fill(selector, text);
-    } else {
-      return console.error("An incorrect value of the input field has been provided");
-    }
-  }
-
```

Comment:
Removed this method from contacts.ts based on the logic - currently this method is more universal and not only relates to the contact page functionality.